test/helpers/fake-project.js

```diff
@@ -87,16 +87,50 @@ module.exports = class FakeProject extends FixturifyProject {
     this.writeSync();
   }
 
-  writeTodoConfig(todoConfig) {
+  setLegacyPackageJsonTodoConfig(daysToDecay) {
     this.pkg = Object.assign({}, this.pkg, {
       lintTodo: {
-        daysToDecay: todoConfig,
+        daysToDecay,
       },
     });
 
     this.writeSync();
   }
 
+  setPackageJsonTodoConfig(engine, daysToDecay, daysToDecayByRule) {
```

Comment:
praise: I like these new naming conventions, it makes everything a little easier for me to reason about.

Reply:
Agree, consistency is good.